crux.el

```diff
@@ -412,6 +412,7 @@ buffer is not visiting a file."
 
 Meant to be used as `find-file-hook'.
 See also `crux-reopen-as-root-mode'."
+  (interactive)
```

Comment:
That function is not meant to be used interactively. There's `crux-sudo-edit` for this purpose.

Reply:
Ah, ok I'll remove the commit